modules/caddyhttp/reverseproxy/fastcgi/fastcgi.go

```diff
@@ -344,6 +345,16 @@ func (t Transport) splitPos(path string) int {
 	return -1
 }
 
+// envVars is a simple type alias to allow for speeding up zap log encoding.
```

Comment:
It's a new type but not an alias. [Aliases](https://go.dev/ref/spec#Alias_declarations) are something different.
```suggestion
// envVars is a simple type to allow for speeding up zap log encoding.
```